pkg/apps/apis/apps/validation/validation.go

```diff
@@ -250,7 +251,12 @@ func validateDeploymentStrategy(strategy *appsapi.DeploymentStrategy, pod *kapi.
 		errs = append(errs, validation.ValidateAnnotations(strategy.Annotations, fldPath.Child("annotations"))...)
 	}
 
-	errs = append(errs, validation.ValidateResourceRequirements(&strategy.Resources, fldPath.Child("resources"), kapivalidation.PodValidationOptions{})...)
+	podClaimNames := sets.NewString()
```

Comment:
@sanchezl Please double check. Thanks.

Reply:
```suggestion
		if len(podResourceClaim.Name) > 0 {
			podClaimNames.Insert(podResourceClaim.Name)
		}
```
To mirror how they build this list upstream.

Reply:
Can you link to the upstream code? This diff confuses me. The `Name` field is not an array.